packages/datagateway-dataview/public/res/default.json

```diff
@@ -100,6 +100,7 @@
     }
   },
   "datasets": {
+    "dataset": "Dataset",
```

Comment:
Decided to add a new field here instead of simply replacing "Name". The name field is used across the dataview and is most useful when sorting or filtering data (e.g. sorting by dataset name). By changing "name" to "Dataset", the column is also called "Dataset" but then sorting and filtering by "Dataset" doesn't really make sense. So I defined a new field to just be used in the investigation landing page.